tests/test_wave_03.py

```diff
@@ -55,12 +55,13 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
+    assert FANTASY_4 in friends_unique_movies
 
```

Comment:
Just to make sure everything was added to the correct list, we can also check to see if HORROR_1 and FANTASY_4 are also in friends_unique_movies.